### DISCUSSION TITLE
Ability to rewind live TV

### DISCUSSION THREAD

**Discussion by alexmick** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1300] [Q&A]
Not sure if it's a change on Catch-Up-TV-and-More or the broadcaster's side but the ability to rewind the live TV feed is gone. I'm pretty sure a few weeks back I could watch FR-TMC or FR-M6 live and rewind back at least 30 minutes to the beginning of the programme. This is great to watch something as it's broadcasting but not yet available on replay.
Does anyone know if this behaviour can be recovered ?

**Comment by ahdiddum** [comment]
Hello,
I do agree with alexmick, rewind feature doesn't work anymore on Catch-Up TV & More, please help !

**Comment by joaopa00** [comment]
Broadcaster stops to send large file with many programs. It sends only small chunks of programs. There is nothing Catchuptvandmore can do.
